pkg/prom/instance/instance_test.go

```diff
@@ -91,11 +94,21 @@ func TestConfig_ApplyDefaults_Validations(t *testing.T) {
 			func(c *Config) { c.ScrapeConfigs[0] = nil },
 			fmt.Errorf("empty or null scrape config section"),
 		},
+		{
+			"missing wal truncate frequency",
+			func(c *Config) { c.WALTruncateFrequency = 0 },
+			fmt.Errorf("wal_truncate_frequency must be greater than 0s"),
+		},
```

Comment:
Add a test for ReturnFlushDeadline > 0 too.

Reply:
whoops, thanks